examples/web/assets/css/app.css

```diff
@@ -0,0 +1,75 @@
+html {
+    position: relative;
+    min-height: 100%;
+}
+
+body {
+    overflow-y: scroll;
```

Comment:
I'm always showing the vertical scrollbar to avoid tab/pill shifting when you switch from a pill with less vertical content.